src/cov/cov-report.c

```diff
@@ -1696,7 +1717,7 @@ void cover_report(const char *path, cover_data_t *data, int item_limit)
    notef("Code coverage report folder: %s.", path);
    notef("%s", tb_get(tb));
 
-   char *top LOCAL = xasprintf("%s/index.html", path);
+   char *top LOCAL = xasprintf("%s%sindex.html", path, DIR_SEP);
```

Comment:
Leave it as `/`. The C library on Windows accepts both.

Reply:
I see, the docs for `fopen` do clarify it. Sorry for the HTML, I did not realize that.
What about the `make_dir`. The underlying `CreateDirectory` docs I was able to find
only mention that `/` can be used on Unix systems.